packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -448,7 +448,7 @@ export const guides = [
       {
         title: "Deploy to Production",
         subLinks: [
-          { title: "Deployment checklist", guideName: "production-deployment" },
+          { title: "Deployment checklists", path: "/docs/guides/deployment-checklist/main/" },
```

Comment:
Question: why are we not using "guideName" and the guide folder? (I'm curious. I'm not sure what the difference is between "path" and "guideName" options?)

Reply:
I'm not 100% sure either. `path` just links the nav item to the path specified, in a simple way. `guideName` seems to do a bunch of automagical stuff, like grabbing the title metadata from the guide's `index.md` file and using that as the link text, and automatically creating all of the sub-menu links in the case of a multi-page guide.

But I found it really tricky to get `guideName` to work for single page articles. It constantly seemed to do weird things, like turning the article title link into a directory with an open/close arrow that just opened to reveal nothing. So I just ended up usinh `path` instead, because it worked more reliably.